app/models.py

```diff
@@ -1293,16 +1293,28 @@ def is_expired(self):
         return self.expired < arrow.now()
 
 
-def generate_email(
+def available_sl_email(email: str, check_trash: bool = True) -> bool:
```

Comment:
It seems that no caller overrides the `check_trash` param, can we remove it?

Reply:
I've removed it. I used in some code that's no longer there.